### ISSUE TITLE
Use NPM instead of Yarn

### ISSUE THREAD

**Issue by ry** [denoland/vscode_deno#148]
The "vsce" program seems to depend on "npm install" having been run. Otherwise we see this:
```
Run npx vsce publish --pat $AZURE_PAT
npx: installed 67 in 2.507s
Executing prepublish script 'npm run vscode:prepublish'...

> vscode-deno@2.0.12 vscode:prepublish /home/runner/work/vscode_deno/vscode_deno
> yarn compile

yarn run v1.22.4
warning vscode-deno@2.0.12: The engine "vscode" appears to be invalid.
$ tsc -b
Done in 16.65s.
 ERROR  Command failed: npm list --production --parseable --depth=99999 --loglevel=error
npm ERR! invalid: typescript-deno-plugin@0.1.0 /home/runner/work/vscode_deno/vscode_deno/node_modules/typescript-deno-plugin
```

So replace all usages of yarn with npm
- yarn.lock with package-lock.json
- github actions should use npm
- package.json scripts should use npm

**Comment by brandonkal** [comment]
Perhaps vsce already works with `yarn dlx` https://yarnpkg.com/cli/dlx

**Comment by lucacasonato** [comment]
Closed in #159
